include/xtensor/xaccessible.hpp

```diff
@@ -186,7 +186,8 @@ namespace xt
 
     template <class D>
     template <class I>
-    inline auto xconst_accessible<D>::operator[](std::initializer_list<I> index) const -> const_reference
+    inline auto xconst_accessible<D>::operator[](std::initializer_list<I> index) const
+        -> const_reference
```

Comment:
I find it less pleasant to read than a single line (screens are wide enough to get it on a single line)